config/initializers/assets.rb

```diff
@@ -6,8 +6,14 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
-Rails.application.config.assets.precompile += %w( shadowbox.css )
-Rails.application.config.assets.precompile += %w( SortedList.js )
```

Comment:
shadowbox.css has been renamed to shadowbox.css.erb. Line 20 has that file. I'm not sure why you get that error. But I put that line back in.